static/images/favicon/site.webmanifest

```
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
+    "icons": [
+        {
+            "src": "/images/android-chrome-192x192.png",
```

I think the proper link is 
```suggestion
            "src": "/images/favicon/android-chrome-192x192.png",
```